.github/workflows/flux-update-scheduled-check.yml

```diff
@@ -0,0 +1,28 @@
+name: Flux Update Schedule Check
+on:
+  schedule:
+    - cron:  '0 12 * * SUN'
+
+jobs:
+  flex-schedule-check:
+    name: Flux Update Schedule Check
+    runs-on:
+      - self-hosted
+      - small
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - uses: webfactory/ssh-agent@v0.7.0
+        with:
+          ssh-private-key: ${{ secrets.PRIVATE_SSH_KEY }}
+
+      - name: Setup asdf
+        uses: asdf-vm/actions/setup@v2
+        env:
+          GITHUB_API_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+
+      - name: Run tests
+        env:
+          GH_TOKEN: ${{ secrets.MESOSPHERECI_USER_TOKEN }}
+        run: make KOMMANDER_REPO_PATH=${GITHUB_WORKSPACE}/kommander make flux-update
```

Comment:
is there an example run for this? 👀

Reply:
https://github.com/mesosphere/kommander-applications/actions/runs/6299669767/job/17100858663